views/user.py

```diff
@@ -0,0 +1,102 @@
+from flask import request
+from flask_restful import Resource
+from models import User
+from http.client import BAD_REQUEST
+from http.client import CREATED
+from http.client import INTERNAL_SERVER_ERROR
+from http.client import NO_CONTENT
+from http.client import NOT_FOUND
+from http.client import OK
+
+
+def non_empty_str(val, name):
+    """ Custom type for reqparser, blocking empty strings. """
+    if not str(val).strip():
+        raise ValueError('The argument {} is not empty'.format(name))
+    return str(val)
+
+
+def email_exists(email):
+    """
+    Check that an email exists in the User table.
+    returns: bool
+    """
+    user = User.select().where(User.email == email)
```

Comment:
Chiaramente se usi la get() non usi la exists(). La mia preferenza è per usare la get:
```
try:
    User.get(User.email == email)
    return True
except User.DoesNotExist:
    return False
```
fondamentalmente perché rispecchia maggiormente il principio EAFP che Python usa (in realtà con alcuni database la get() è più performante perché fa un limit(1) che rappresenta una leggera ottimizzazione)